README.md

```diff
@@ -1,159 +1,34 @@
-<img src=".erb/img/erb-banner.svg" width="100%" />
+## Draft-Craft
 
-<br>
+This project is a notes app built using React and Electron. The app features a simple, user-friendly interface designed for productivity and note management.
 
-<p>
-  Electron React Boilerplate uses <a href="https://electron.atom.io/">Electron</a>, <a href="https://facebook.github.io/react/">React</a>, <a href="https://github.com/reactjs/react-router">React Router</a>, <a href="https://webpack.js.org/">Webpack</a> and <a href="https://www.npmjs.com/package/react-refresh">React Fast Refresh</a>.
-</p>
+### Setting Up the Project
 
-<br>
+## Prerequisites
```

Comment:
```suggestion
### Prerequisites
```
This heading comes under `Setting Up the Project`. The heading style should be one below of it.